mysqld_exporter.go

```diff
@@ -49,9 +53,27 @@ const (
 	namespace = "mysql"
 	// Subsystems.
 	exporter          = "exporter"
-	slaveStatus       = "slave_status"
 	globalStatus      = "global_status"
+	informationSchema = "info_schema"
 	performanceSchema = "perf_schema"
+	slaveStatus       = "slave_status"
+)
+
+// Metric SQL Queries.
+const (
+	infoSchemaAutoIncrementQuery = `
+SELECT table_schema, table_name, column_name, auto_increment,
```

Comment:
Why not indent one tab more than the `infoSchemaAutoIncrementQuery`? Makes indentation levels visually clearer.